tokio/Cargo.toml

```diff
@@ -120,6 +120,9 @@ version = "0.3.8"
 default-features = false
 optional = true
 
+[target.'cfg(tokio_unstable)'.dependencies]
```

Comment:
Let's remove the `tokio_unstable` flag. We need more people using this and there are upcoming breaking changes (tokio 0.3) planned.